dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.ws.ejbcontainer.security.jacc_fat.2.1</name>
```

Comment:
Ideally this would be in an io.openliberty prefixed project since it is a new project.  It is weird that it is called 2.1 when it tests other versions besides 2.1 it appears.